assets/config.php

```diff
@@ -35,6 +35,37 @@
         // Stancl\Tenancy\Bootstrappers\RedisTenancyBootstrapper::class, // Note: phpredis is needed
     ],
 
+
+    /**
+     * Pending tenancy config.
+     * This is useful if you're looking for a way to always have a tenant ready to be used.
+     */
+    'pending' => [
+        /**
+         * If disabled, pending tenants will be excluded from all tenant queries. Unless if
+         * told otherwise with ::withPending() or ::onlyPending().
+         * Note: when disabled, this will also ignore tenants when runnings any tenants commands (migration, seed, etc.)
```

Comment:
- The `told otherwise` line is weird, but keep the examples (`::withPending()` and `::onlyPending()`)
- `runnings` in second line, general grammar

Reply:
I updated the comments regarding the pending tenants in the config file

https://github.com/archtechx/tenancy/pull/869/commits/83f2cab9449dad30ff523050fb2f44af143c34e4

Reply:
Something's up with the CI tests – after pushing the comment updates, there's this error:
<img width="886" alt="image" src="https://user-images.githubusercontent.com/34375937/180771525-dcaf629b-a3be-49aa-9d30-2364353d6456.png">